web/src/main/resources/application.properties

```diff
@@ -2,10 +2,6 @@
 spring.jackson.date-format=com.microsoft.azure.simpletodo.configuration.RFC3339DateFormat
 spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
 
-spring.datasource.url=${AZURE_POSTGRESQL_URL}
-spring.datasource.username=${AZURE_POSTGRESQL_USERNAME}
-spring.datasource.azure.passwordless-enabled=true
-
-spring.sql.init.mode=always
-spring.sql.init.schema-locations=classpath*:schema-all.sql
-spring.sql.init.username=${AZURE_POSTGRESQL_USERNAME}
+spring.jpa.generate-ddl=true
```

Comment:
If we remove this, the sample can't run in my local, should we mention it in README?